packages/sdk/python/src/relayfile/integration_adapter.py

```
@@ -62,9 +62,69 @@ class SyncResult:
+
+
+_DEFAULT_PRESENT_TEXT_LIMIT = 500
+
+
+def generic_present(event: WriteEvent, adapter: IntegrationAdapter) -> RelayBindingView:
+    provider = _provider_from_path(event.path) or adapter.name
+    payload: dict[str, Any] = event.value if isinstance(event.value, dict) else {}
+    try:
+        object_type, object_id = _object_identity_from_path(event.path)
+        semantics = adapter.compute_semantics(object_type, object_id, payload)
+    except Exception:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)